.travis.yml

```diff
@@ -1,2 +1,10 @@
 language: rust
 rust: nightly
+cache: cargo
+before_script:
+- export PATH="$PATH:$HOME/.cargo/bin"
+- which rustfmt || cargo install rustfmt
+script:
+- cargo fmt -- --write-mode=diff
```

Comment:
Yes (I'm not sure whether this is desirable yet, I'm just copying the advice here: https://github.com/rust-lang-nursery/rustfmt)